src/Microsoft.OData.ModelBuilder/Microsoft.OData.ModelBuilder.xml

```diff
@@ -5106,5 +5116,2153 @@
+            Change tracking supports filters on these properties
+            If no properties are specified or FilterableProperties is omitted, clients cannot assume support for filtering on any properties in combination with change tracking.
+            </summary>
+            <param name="filterableProperties">The value(s) to set</param>
+            <returns><see cref="T:Microsoft.OData.ModelBuilder.Capabilities.V1.ChangeTrackingConfiguration"/></returns>
+        </member>
+        <member name="M:Microsoft.OData.ModelBuilder.Capabilities.V1.ChangeTrackingConfiguration.AddExpandableProperties(Microsoft.OData.Edm.Vocabularies.EdmNavigationPropertyPathExpression[])">
+            <summary>
+            Change tracking supports these properties expanded
+            If no properties are specified or ExpandableProperties is omitted, clients cannot assume support for expanding any properties in combination with change tracking.
+            </summary>
+            <param name="expandableProperties">The value(s) to set</param>
+            <returns><see cref="T:Microsoft.OData.ModelBuilder.Capabilities.V1.ChangeTrackingConfiguration"/></returns>
+        </member>
+        <member name="M:Microsoft.OData.ModelBuilder.Capabilities.V1.ChangeTrackingConfiguration.ToEdmExpression">
+            <inheritdoc/>
+        </member>
+        <member name="T:Microsoft.OData.ModelBuilder.Capabilities.V1.CollectionPropertyRestrictionsConfiguration">
+            <summary>
+            Describes restrictions on operations applied to collection-valued structural properties
+            </summary>
+        </member>
+        <member name="M:Microsoft.OData.ModelBuilder.Capabilities.V1.CollectionPropertyRestrictionsConfiguration.#ctor">
+            <summary>
+            Creates a new instance of <see cref="T:Microsoft.OData.ModelBuilder.Capabilities.V1.CollectionPropertyRestrictionsConfiguration"/>
+            </summary>
+        </member>
+        <member name="M:Microsoft.OData.ModelBuilder.Capabilities.V1.CollectionPropertyRestrictionsConfiguration.AddCollectionPropertyRestrictions(Microsoft.OData.ModelBuilder.Capabilities.V1.CollectionPropertyRestrictionsTypeConfiguration[])">
+            <summary>
+            Describes restrictions on operations applied to collection-valued structural properties
+            </summary>
+            <param name="collectionPropertyRestrictions">The value(s) to set</param>
+            <returns><see cref="T:Microsoft.OData.ModelBuilder.Capabilities.V1.CollectionPropertyRestrictionsConfiguration"/></returns>
+        </member>
+        <member name="M:Microsoft.OData.ModelBuilder.Capabilities.V1.CollectionPropertyRestrictionsConfiguration.ToEdmExpression">
+            <inheritdoc/>
+        </member>
+        <member name="T:Microsoft.OData.ModelBuilder.Capabilities.V1.CollectionPropertyRestrictionsTypeConfiguration">
+            <summary>
+            Org.OData.Capabilities.V1.CollectionPropertyRestrictionsType
```

Comment:
?

Reply:
This type lacked a description annotation, I fall back to the  full type name... Not sure of a more appropriate alternative.